### DISCUSSION TITLE
Craft 4 support?

### DISCUSSION THREAD

**Discussion by amphibian** [trendyminds/craft-reporter#4] [Q&A] [answered]
I have yet to find an export plugin as flexible as Reporter. Any chance Craft 4 support is coming?

**Comment by aaronbushnell** [comment, marked as answer]
I'm (very) late on this, but Craft 4 support is available after an update we made in June 2023! https://github.com/trendyminds/craft-reporter/blob/main/CHANGELOG.md#200---2023-06-22
